pkg/resource/fake/mocks.go

```diff
@@ -249,7 +249,11 @@ func (m *EnvironmentConfigReferencer) GetEnvironmentConfigReferences() []corev1.
 
 // ConnectionDetailsLastPublishedTimer is a mock that implements the
 // ConnectionDetailsLastPublishedTimer interface.
-type ConnectionDetailsLastPublishedTimer struct{ Time *metav1.Time }
+type ConnectionDetailsLastPublishedTimer struct {
+	// runtime.DefaultUnstructuredConverter.ToUnstructured cannot currently
```

Comment:
nit: I think this should be a note rather than comment of the field.

Reply:
Done.